scripts/generate_crds_manifests.sh

```diff
@@ -88,12 +88,12 @@ add_ibm_managed_cloud_annotations() {
 }
 
 ${YQ} merge --inplace -d'*' manifests/0000_50_olm_00-namespace.yaml scripts/namespaces.patch.yaml
-${YQ} merge --inplace --arrays=append -d'1' manifests/0000_50_olm_01-olm-operator.serviceaccount.yaml scripts/olm-service.patch.yaml
+${YQ} merge --inplace --arrays=append -d'1' manifests/0000_50_olm_*-olm-operator.serviceaccount.yaml scripts/olm-service.patch.yaml
```

Comment:
```suggestion
${YQ} merge --inplace --arrays=append -d'1' manifests/0000_50_olm_[0-9][0-9]-olm-operator.serviceaccount.yaml scripts/olm-service.patch.yaml
```
Seems safer to limit this to require the digits, otherwise maybe glob catches more than desired in a hard to figure out why type of way.

Reply:
Fixed it! Nice catch!!